docs/docs/reference/crd-reference/config.md

```diff
@@ -17,6 +17,7 @@ spec:
   OTelCollectorUrl: '<otelurl:port>'
   keptnAppCreationRequestTimeoutSeconds: <#-seconds>
   cloudEventsEndpoint: <endpoint>
+  blockDeployment: <bool>
```

Comment:
What would you think about showing the value as "true | false" rather than "<bool>"?  I can see arguments for each option, I suppose.